File: JavaPractice/src/FizzBuzz.java
/**
 * @author divyesh.shah
 */
public class FizzBuzz {
    public static void main(String[] args) {
        int n = 100; // set the range of integers to 1 to 100 (inclusive)

        for (int i = 1; i <= n; i++) {
            if (i % 3 == 0 && i % 5 == 0) {
                System.out.println("FizzBuzz");
            } else if (i % 3 == 0) {
                System.out.println("Fizz");
            } else if (i % 5 == 0) {
                System.out.println("Buzz");
            } else {
                System.out.println(i);
            }
        }
    }
}

File: PythonPractice/src/FizzBuzz.py
def fizz_buzz(n: int) -> None:
    if not isinstance(n, int) or n <= 0:
        raise ValueError("Input must be a positive integer.")

    for i in range(1, n + 1):
        if i % 3 == 0 and i % 5 == 0:
            print("FizzBuzz")
        elif i % 3 == 0:
            print("Fizz")
        elif i % 5 == 0:
            print("Buzz")
        else:
            print(i)

if __name__ == "__main__":
    n = 100  # set the range of integers to 1 to 100 (inclusive)
    fizz_buzz(n)
